cell_type_prediction/README.md

```diff
@@ -56,5 +56,49 @@ Two output tables are produced, the first containing a list of all predicted lab
 4) Semantic similarity across predicted labels. Correctly predicted cells are likely to have labels that are highly similar among them (think high precision). 
 5) Aggregated score: again, scores  2-4 are averaged to produce combined score. Cells are sorted by this score. Those cells that have low score are less likely to be correctly predicted. 
 
+## Use case: run cell type predictions programmatically
+It is possible to run the workflow programmatically using the [galaxy-workflow-executor](https://github.com/ebi-gene-expression-group/galaxy-workflow-executor) package. For more detail, see the README on the provided repository. You will need [conda](https://docs.conda.io/en/latest/) to install it. Run the following commands: 
+
+```
+conda create -n <env_name> galaxy-workflow-executor
+conda activate <env_name>
+```
+
+The following steps will walk you through the procedure. Example data is provided in [galaxy_cli_use_case](galaxy_cli_use_case) directory.
+ - To generate the workflow parameters yaml file, run 
+ ```
+ generate_params_from_workflow.py -C galaxy_credentials.yaml \
+                            -G test_instance -o test \
+                            -W cell_types_prod_control_wf.ga
+ ```
+
+The above script will generate a [parameters file](galaxy_cli_use_case/cell_types_prod_control_wf_parameters.yaml).
+
+- You will need to import the following files:
+   
```

Comment:
Have you considered adding a step in the workflow to retrieve these? There is currently a galaxy module that gets these from the ftp. Then from the outside you would just specify the Atlas accessions. Is this for the indexes generation or for the query to be used? I would expect the query to be user provided matrices, right?

Reply:
@pcm32 This is just an example of a user-provided data, so we can't automate that. Everything required for prediction is retrieved from the FTP server.

Reply:
..ok, fair enough, so you are using the Atlas data as it would be the user provided one, I understand.